controllers/csm_controller.go

```diff
@@ -1279,19 +1276,14 @@ func (r *ContainerStorageModuleReconciler) PreChecks(ctx context.Context, cr *cs
 		}
 	default:
 		for _, m := range cr.Spec.Modules {
-			if m.Name == csmv1.AuthorizationServer {
-				return nil
-			}
-			if m.Name == csmv1.ApplicationMobility {
-				am = true
+			if m.Name == csmv1.ApplicationMobility || m.Name == csmv1.AuthorizationServer {
```

Comment:
Instead of a `goto checkUpgrade`, can we use a conditional variable called something like `isStandaloneMod` that we initialize to false, set to true in the for loop if we find either app-mobility or auth server, and then check at line 1283?